src/script/common/c_content.cpp

```diff
@@ -191,7 +191,7 @@ void read_object_properties(lua_State *L, int index,
 
 	int hp_max = 0;
 	if (getintfield(L, -1, "hp_max", hp_max))
-		prop->hp_max = (s16)rangelim(hp_max, 0, S16_MAX);
+		prop->hp_max = rangelim(hp_max, 0, S16_MAX);
```

Comment:
shouldn't this be a template too?

Reply:
`prop->hp_max = rangelim<s16>(hp_max);`

Reply:
@sfan5 `rangelim` here is the template declared in `numeric.h` You don’t need to use `<>` to use a template *function* (as opposed to template *class*).